nextflow_schema.json

```diff
@@ -309,7 +309,7 @@
                 },
```

Comment:
the other params don't need to be updated here?

Reply:
No, all others were good already because of the json schema

Reply:
issues where in the nextflow.config